keras_retinanet/models/vgg.py

```diff
@@ -0,0 +1,72 @@
+"""
+Copyright 2017-2018 cgratie (https://github.com/cgratie/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+import keras
+
+from ..models import retinanet
+
+
+custom_objects = retinanet.custom_objects.copy()
+
+
+def download_imagenet(backbone):
+    backbone = int(backbone.replace('vgg', ''))
```

Comment:
Why not keep it as string?

Reply:
I just used existing code as an example, but you are right, string would work better in this case.